README.md

```diff
@@ -26,7 +26,7 @@ kubectl apply -f - <<EOF
 apiVersion: opentelemetry.io/v1alpha1
 kind: OpenTelemetryCollector
 metadata:
-  name: simplest
+  name: otel
```

Comment:
renaming this to otel. Simplest creates `simplest-collector` and otel creates `otel-collector` service which is more idiomatic for an example.

Reply:
That doesn't seem right to me for two reasons:

- `otel` != `otelcol`. An OpenTelemetry Collector (otelcol) CR named just "OpenTelemetry" (otel) is technically wrong. How about OpenTelemetry Instrumentations? Shouldn't they also be called `otel` then?
- naming the CR (`otel`) after one of the resources it creates (`otel-collector`) doesn't look like a good practice

Besides, as someone starting with this project, I think it's clearer to understand the effects of my actions if I use a name that isn't related to the underlying tech. If I create a CR named `simplest` and see a `simplest-collector-headless`, I know immediately that this is specific to my service. If I just named it `otel`, I could miss the fact that `otel-collector-headless` is specific to my service, as opposed to something the operator itself needs.

Reply:
`simplest` creates (and other objects named similarly) - `service/simplest-collector` and `deployment/simplest-collectors`. Now, I fear that this is not expressive enough and might collide with other objects in a namespace. Users should know what technology they are deploying, bc then it depends on what they can integrate it with or connect to it. 

> otel != otelcol.

yes, but the created objects in the cluster make total sense to me `deployment.apps/otel-collector`, `service/otel-collector`.  Now in the instrumentation or envoy I will use `otel-collector:4317` which is idiomatic enough to understand what it is instead of `simplest-collector:4317` (what is simplest? Is it Jaeger, Zipkin or my simple demo from the last kubecon?) 


That said the name in the ALM examples is also `otel`  https://github.com/open-telemetry/opentelemetry-operator/blob/main/bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml#L23.

Reply:
> That said the name in the ALM examples is also otel https://github.com/open-telemetry/opentelemetry-operator/blob/main/bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml#L23.

Apparently you changed it in #519 ;-)

Reply:
yes bc the name was `opentelemetrycollector-sample` and it created objects named `opentelemetrycollector-sample-collector`

Reply:
My point remains: `otel` is a bad example name for this CR, despite the existence of an example with that name.

Reply:
For instance I don't care about the instrumentation CR name because it does not create any objects. Which is not the case for `OpenTelemetryCollector` CR. If the CR creates objects I do care more about the object names and especially about service, deployment and pod names and not that much about the CR name. The service,deployment,pod names are the most important this is what people see and relate to.

Reply:
I am open to suggestions :)

Reply:
> I am open to suggestions :)

Don't change it ;-)

Reply:
I really don't like `simplest-collector` it is not idiomatic and when I see it in my cluster I literally don't know what it is!

Would you name a MongoDB instance `simple-database`?

Reply:
I am curious what instance name of oTEL collector are you using in grafana.

Reply:
nevermind I have renamed `otel` back to `simplest` even though I find it ambiguous for new users.